index.ts

```diff
@@ -0,0 +1,72 @@
+import { spawn, SpawnOptionsWithoutStdio } from "child_process";
+import type { Plugin as VitePlugin } from "vite";
+
+// Utility to invoke a given sbt task and fetch its output
+function printSbtTask(task: string, cwd?: string): Promise<string> {
+  const args = ["--batch", "-no-colors", "-Dsbt.supershell=false", `print ${task}`];
+  const options: SpawnOptionsWithoutStdio = {
+    cwd: cwd,
```

Comment:
consider: `stdio: { 'ignore', 'pipe', 'inherit' }`. You can then drop the `on` handler below for stderr.